### PR TITLE
Adding USGS review

### PR DESCRIPTION
**Pull request by mnfienen**
I tried to address all USGS review comments. 

Need to sort out referencing the figure in the text and also make sure the bulleted lists render right at the JOSE site.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -82,30 +82,30 @@ The tutorial notebooks discussed herein provide a comprehensive, self-guided, an
...
 
-GMDSI is dedicated to promoting, facilitating, and providing support for the improved utilization of modeling in decision support processes. Its activities endeavor to elevate the role of groundwater modeling in decision-making processes, recognizing the importance of model partner software  for UQ and PE, and offering a range of activities aimed at industry engagement, education, practical examples, research, and software development.
+GMDSI is dedicated to promoting, facilitating, and providing support for the improved utilization of modeling in decision support processes. Its activities endeavor to elevate the role of groundwater modeling in decision-making processes, recognizing the importance of model partner softwar. for UQ and PE, and offering a range of activities aimed at industry engagement, education, practical examples, research, and software development.
```
Comment (jtwhite79) [inline paper/paper.md:85]: Did you mean for this to change...

File: paper/paper.md [hunk 2]
```diff
@@ -82,30 +82,30 @@ The tutorial notebooks discussed herein provide a comprehensive, self-guided, an
...
 
 # Resources
 
 A webinar hosted by GMDSI introducing the tutorial notebooks can be viewed [here](https://vimeo.com/856752189). During the webinar the authors provided an overview of the notebooks, as well as a demonstration of how to use them and introduced an [online self-guided course](https://gmdsi.org/blog/guided-self-study-course/).
 
-The [GMDSI](https://gmdsi.org) web-page also hosts an extensive range of resources  and educational material on decision support modelling. These include numerous instructional video lectures, webinar recordings, non-programmatic workflow tutorials, as well as worked example reports describing real-world applications. 
+The [GMDSI](https://gmdsi.org) web-page also hosts an extensive range of resource. and educational material on decision support modelling. These include numerous instructional video lectures, webinar recordings, non-programmatic workflow tutorials, as well as worked example reports describing real-world applications.
```
Comment (jtwhite79) [inline paper/paper.md:102]: Is this "resource." what you meant?

File: paper/paper.md (hunk 1 above)
Comment (mnfienen) [reply paper/paper.md:85]: this seems like a merge issue

File: paper/paper.md (hunk 2 above)
Comment (mnfienen) [reply paper/paper.md:102]: this looks like a goof-up from merging too. I think it was good before

**Review by rhugman — APPROVED** [review]

**Pull request merged**